src/components/examples/ReadOnly.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Demo } from '../common';
+
+const data = {
+  firstname: "Max",
+  lastname: "Mustermann"
+}
+
+const readonlyConfig = {
+  schema: {
+    properties: {
+      firstname: {
```

Comment:
We should use `firstName` and `lastName` for all properties so that the labels look better